Sources/web3swift/Tokens/ERC20/Web3+ERC20.swift

```diff
@@ -80,12 +79,12 @@ public class ERC20: IERC20, ERC20BaseProperties {
         guard let value = Utilities.parseToBigUInt(amount, decimals: intDecimals) else {
             throw Web3Error.inputError(desc: "Can not parse inputted amount")
         }
+        contract.transaction = transaction
```

Comment:
Why don't we do the same in all of the functions left?
For example, check `public func setAllowance`.  It looks almost identical to `public func transfer` in terms of the logic of each step inside the function: set transaction `from`, `to` and `callOnBlock `; call read-only `"decimals"` function; create `WriteOperation`.

Reply:
This one is actually a but that I've left. It's fixed just yet.